account_budget_activity/models/account_budget.py

```diff
@@ -702,48 +704,37 @@ def _compute_planned_amount(self):
             rec.planned_amount = planned_amount  # from last year
 
     @api.multi
-    @api.depends('r1', 'r2', 'r3', 'r4', 'r5', 'r6',
-                 'r7', 'r8', 'r9', 'r10', 'r11', 'r12', )
```

Comment:
@mra-odoo if the r1 - r12 is not used anymore, please remove all of them from code.